views/partials/fxa_enabled/fxa_monitor.hbs

```diff
@@ -0,0 +1,36 @@
+<main class="landing-content">
+  <div class="section-wrapper">
+    <div class="half {{#if featuredBreach}} featured {{/if}}">
+      <h2 data-location="landing-page" class="section-headline">
+        {{#if featuredBreach }}
+          {{{fluentFormat req.supportedLocales "scan-featuredbreach-label" featuredBreach=featuredBreach.Title}}}
+        {{else}}
+          {{{fluentFormat req.supportedLocales "landing-headline"}}}
```

Comment:
General observation, but I'm a bit confused about the usage of `{{` vs `{{{` in these templates.

Looking at the .ftl file, `landing-headline` is just a simple string:

```ini
landing-headline = Your right to be safe from hackers starts here.
```

Do we need the triple curly braces here since we're not displaying any nested HTML content?